telemetry/csharp/AwsToolkit.Telemetry.Events.Tests/Generated/GeneratedCode.cs

```diff
@@ -6147,11 +6332,21 @@ public sealed class AwsLoadCredentials : BaseTelemetryEvent
         
         /// The ID of the source of credentials (e.g. profile)
         public string CredentialSourceId;
+        
+        public AwsLoadCredentials()
+        {
+            this.Passive = true;
+        }
     }
     
     /// Create a new credentials file
     public sealed class AwsCreateCredentials : BaseTelemetryEvent
     {
+        
+        public AwsCreateCredentials()
+        {
+            this.Passive = false;
```

Comment:
nit: why is this not an argument to `BaseTelemetryEvent`?

Reply:
Currently the code-generation code is very brute force. It would have been more complicated to have the code generator produce code that passes this to a base class constructor, but is a good candidate for a future improvement.